### ISSUE TITLE
Wrong .gitignore addition

### ISSUE THREAD

**Issue by bobvandevijver** [symfony/webpack-encore#286]
Not sure what happened, but after installing encore (with `php composer.phar require encore`), the `.gitignore` was appended with the webpack-encore-pack information, but the public dir was not entered correctly. 

I would expect it to be filled correctly. Is this is a bug, or are we expected to change this manually?

The change:
```gitignore
###> symfony/webpack-encore-pack ###
/node_modules/
/%PUBLIC_DIR%/build/
###< symfony/webpack-encore-pack ###
```

**Comment by Lyrkan** [comment]
Hi @bobvandevijver,

I'm going to close this issue since it should probably be reported on [the flex repository](https://github.com/symfony/flex/issues) instead of that one but for what it's worth I can't reproduce it on my side:

```
$ composer create-project symfony/skeleton
Installing symfony/skeleton (v4.0.5)
  - Installing symfony/skeleton (v4.0.5)
    Downloading: 100%  
(...)

$ cd skeleton

$ composer require encore
Using version ^1.0 for symfony/webpack-encore-pack
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Installing symfony/webpack-encore-pack (v1.0.2)
(...)

$ cat .gitignore

###> symfony/framework-bundle ###
/.env
/public/bundles/
/var/
/vendor/
###< symfony/framework-bundle ###

###> symfony/webpack-encore-pack ###
/node_modules/
/public/build/
###< symfony/webpack-encore-pack ###
```

Maybe it could be related to the version of flex you are using.

**Comment by bobvandevijver** [comment]
You're right. Just checked the version of flex, updated it from v1.0.70 to v1.0.76, and now it does work as expected. 

Also found the bug, which was fixed in v1.0.75 (released 4 days ago): https://github.com/symfony/flex/pull/337
